### PR TITLE
files succesfully uploaded

### PR DESCRIPTION
**Pull request by NikitaShivhare1**

### CODE REVIEW

File: Nikita/Assignment_6/assignment_6.html [hunk 1]
```diff
@@ -1,56 +1,59 @@
...
-                <input type="radio" name="g" value="male" />Less Than 18
-                <input type="radio" name="g" value="Female" />18-25
-                <input type="radio" name="g" value="other" />26-40
-            </td>
-        </tr>
+      <tr>
+        <td>First name <input type="text" placeholder="First Name" /></td>
+      </tr>
+      <tr>
+        <td>Last name <input type="text" placeholder="Last Name" /></td>
```
Comment (shubhamdalal49) [inline Nikita/Assignment_6/assignment_6.html]: placeholder second word first letter should be small

**Review by shubhamdalal49 — CHANGES_REQUESTED** [review]

**Review by shubhamdalal49 — APPROVED** [review]

**Review by Ashwin2812 — APPROVED** [review]

File: Nikita/Assignment_5/Assignment5.html [hunk 2]
```diff
@@ -1,27 +1,36 @@
...
+                <li><a href="" target="_blank" class="link">My Hobbies</a></li>
+                <li><a href="" target="_blank" class="link">My Photos</a></li>
             </ul>
-       </div>
-    
\ No newline at end of file
+        </div>
+    </div>
+</body>
+</html>
```
Comment (Pawan1005) [inline Nikita/Assignment_5/Assignment5.html:42]: indentation missing

File: Nikita/Assignment_6/Assignment_6.2.html [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+          <label class="label-comment"><b>Comments:</b></label>
+          <textarea cols="30" rows="10"></textarea>
+        </div>
+      </fieldset>
+      <div class="btn1">
+        <input type="button" value="Submit" />
+      </div>
+    </form>
+  </body>
+</html>
```
Comment (Pawan1005) [inline Nikita/Assignment_6/Assignment_6.2.html:102]: indentation missing

File: Nikita/Assignment_6/assignment_6.html [hunk 4]
```diff
@@ -1,53 +1,59 @@
...
+          <input type="password" />
+        </td>
+      </tr>
     </table>
-
-</body>
-</html>
\ No newline at end of file
+  </body>
+</html>
```
Comment (Pawan1005) [inline Nikita/Assignment_6/assignment_6.html:64]: indentation missing

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

File: Nikita/Assignment_6/assignment_6.html (hunk 4 above)
Comment (Priyankashelke2411) [reply Nikita/Assignment_6/assignment_6.html:64]: give proper indentation

**Review by Priyankashelke2411 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**